### ISSUE TITLE
Get logits and LM loss from the DNABert

### ISSUE THREAD

**Issue by JinyuanSun** [Zhihan1996/DNABERT_2#53]
Hi, I tried to get logits of a input sequence using DNABert, here is my code:
```
tokenizer = AutoTokenizer.from_pretrained("zhihan1996/DNABERT-2-117M", trust_remote_code=True)
model = AutoModel.from_pretrained("zhihan1996/DNABERT-2-117M", trust_remote_code=True)
model.eval().cuda()

seq = "TCCCACTATTTGTCGGCTAGCCAGATTGTTGTGGTCTGATTAAAGTT\
TCAATTTATACCTTACAATGATGTAAGGTACGTGTAAGAGAAATCGATGGGATA\
TTTTTTTACAACAAGGTATTCTTAAAGTAAGAGTTATACGCTATGTGGAAAAGAGGTGTTTAAG"

tokens_ids = tokenizer.batch_encode_plus([seq], return_tensors="pt")["input_ids"]
attention_mask = tokens_ids != tokenizer.pad_token_id

torch_outs = model(tokens_ids.cuda(),
                   attention_mask=attention_mask.cuda(),
                   encoder_attention_mask=attention_mask.cuda(),
                   output_hidden_states=True,
                   labels=tokens_ids.cuda())
```
The torch_outs returned is a tuple having 2 tensors: `torch.Size([1, 39, 768]) and torch.Size([1, 768])`. I assumed the first one are embeddings for each token, and the second is a pooled embeddings of this sequence?

Is that possible for the DNABert to return logits of each token?

**Comment by JinyuanSun** [comment]
A simple solution is to load model through `AutoModelForMaskedLM` instead:
```python
from transformers import AutoTokenizer, AutoModelForMaskedLM

tokenizer = AutoTokenizer.from_pretrained("zhihan1996/DNABERT-2-117M", trust_remote_code=True)
model = AutoModelForMaskedLM.from_pretrained("zhihan1996/DNABERT-2-117M", trust_remote_code=True)

```
the model returns will contain logits and loss.
